src/lib/crypto/kimchi_pasta_snarky_backend/test/test_bigint.ml

```diff
@@ -0,0 +1,19 @@
+open Core_kernel
+
+let test_hex_conversion () =
+  let module Bigint = Kimchi_pasta_snarky_backend.Bigint256 in
+  let length_in_bytes = Bigint.length_in_bytes in
+  let bytes =
+    String.init length_in_bytes ~f:(fun _ -> Char.of_int_exn (Random.int 255))
```

Comment:
maybe this allows some numbers which are not in pallas or vesta, and might want to limit to 254 bits

Reply:
Good point.